test/utils.test.sh

```diff
@@ -390,6 +390,60 @@ assert "contains" "${log}" "pushing test to us-east-1"
 assert "contains" "${log}" "found existing image for test in us-west-2, skipping push"
 assert "equal" "${FAILURE}" "" "should not have any failures"
 
+# bucket_push() no ImageBucketPrefix
+tag_test "bucket_push"
+export ImageBucketPrefix=""
+
+log=$(bucket_push)
+expected="nothing to do"
+assert "equal" "${log}" "${expected}"
+
+# bucket_push()
```

Comment:
maybe a multi region test could be helpful here or some kind of test for how many times `aws` was called? 
For example, if you had one region, could there be some kind of test that asserts `aws s3 cp` was called once?
And if you had multiple regions, i.e. ('us-east-1 us-east-2 us-west-2'), the test would be like `aws s3 cp called three times` ?

Along the lines, if someone entered a comma separated list of regions, would the function fail? It might be helpful to have a failure test for incorrect parameters or possibly even error handling! (I'm really not sure if that's overkill though 😅 )